README.md

```diff
@@ -37,6 +37,44 @@ This package implements functions for:
 1. **Logging**: Steps in the pipeline have comprehensive R-style logging, with the the [cli](https://github.com/r-lib/cli) package
 1. **Metadata**: Extract comprehensive metadata on the model run and store alongside outputs
 
+## Output format
+
+The hybrid partitioning structure of the outputs is designed to facilitate both automated processes and manual investigation: files are organized by job and task IDs, allowing for efficient file operations using glob patterns, while also maintaining a clear hierarchy that aids human users in navigating to specific results or logs. Files meant primarily for machine-readable consumption (i.e., draws, summaries, diagnostics) are structured together to make globbing easier. Files meant primarily for human investigation (i.e., logs, model fit object) are grouped together by task to facilitate manual workflows.
```

Comment:
I don't know what a hybrid partitioning structure is

Reply:
Resolved by https://github.com/CDCgov/cfa-epinow2-pipeline/pull/41/commits/21339d886a07b0d1bdc40d7f0c6a046b8aae1d96